frontend/src/stores/ParticipantStore.js

```diff
@@ -213,9 +213,7 @@ export class ParticipantStore {
       throw `ParticipantStore:  getParticipants() Failed  =>  ${error}`
     }
   })
-  // called on  =>  ParticipantInfo.js
-  // andVisit temporary hack, this logic should be seperated out when creating VisitView
-  createParticipant = flow(function*(andVisit = true) {
+  createParticipant = flow(function*() {
```

Comment:
we separated this logic with the NewParticpantView and ExistingParticipantView

Reply:
@MikeyManoguerra Looks like the 'andVisit' hack was also creating a visit, I'm not seeing that function in either NewParticipantView or ExistingParticipantView, are we just not creating a visit simultaneously with a participant or is that function going to be called elsewhere?

Reply:
This is a relic, related to visits that would end up in a queue. Originally, participant and visit was just one form, so this was an intermediary variable to handle splitting the two forms. now, when you create a participant in the new Participant view, it will route to the existing participant view and display the new participant there. the user then has the option to create a visit at that time